src/krate.rs

```diff
@@ -785,6 +785,27 @@ impl Model for Crate {
 }
 
 /// Handles the `GET /crates` route.
+/// Returns a list of crates. Called in a variety of scenarios in the
+/// front end, including:
+/// - Alphabetical listing of crates
+/// - List of crates under a specific owner
+/// - Listing a user's followed crates
+///
+/// Notes:
+/// The different use cases this function covers is handled through passing
+/// in parameters in the GET request.
+///
+/// This function was has had functionality bolted on over the years, this
```

Comment:
"This function was has had" => I think the "was" can come out :)

"this needs to stop" => I would say "we would like to stop adding more functionality in here" :)